lib/floe/workflow/states/mapped_state.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Floe
+  class Workflow
+    module States
+      class MappedState < Floe::Workflow::State
+        attr_reader :input_path, :output_path
+
+        def initialize(workflow, name, payload)
+          super
+          @input_path  = Path.new(payload.fetch("InputPath", "$"))
+          @output_path = Path.new(payload.fetch("OutputPath", "$"))
+        end
+
+        def run!(input)
+          input = input_path.value(context, input) if input_path
+          output = run_input(input)
+          output = output_path.value(context, output) if output_path
```

Comment:
Do we want to be lenient here?
Or is this a configuration issue and we should let it throw an error?